src/ui/simulator/windows/inspector/frame.cpp

```diff
@@ -180,7 +180,7 @@ void Frame::onSelectAllPlants(wxCommandEvent&)
             Data::Area& area = *(*i);
             auto end = area.thermal.list.end();
```

Comment:
Change into a `for(auto& cluster : area->thermal.list)`, quickly done, and more clear